pkg/scheduler/framework/plugins/volumerestrictions/volume_restrictions.go

```diff
@@ -142,10 +224,22 @@ func (pl *VolumeRestrictions) EventsToRegister() []framework.ClusterEvent {
 		{Resource: framework.Pod, ActionType: framework.Delete},
 		// A new Node may make a pod schedulable.
 		{Resource: framework.Node, ActionType: framework.Add},
+		// Pods may fail to schedule because the PVC it uses has not yet been created.
+		// This PVC is required to exist to check its access modes.
```

Comment:
naive question, can a claim be "reclaimed"? can a PVC be released for another pod to claim? if so, we need to include the PVC update events too.

Reply:
A user could potentially tweak the pointers on the PVC and PV to change the binding, but these changes to the PVC spec are blocked by immutability protection. I will include responding to updates to future-proof should this change.